routes/product.js

```diff
@@ -19,3 +20,13 @@ async function createProduct(req, res) {
     res.status(422).json({ message: 'Could not create product', e });
   }
 }
+
+async function getProducts(req, res) {
+  try {
+    const products = await productModel.findAll();
+    res.json({ products, message: 'sucess' });
+  } catch (e) {
+    console.log('error', e);
+    res.status(422).json({message: 'Failed to fetch data from database', e});
+  }
+}
```

Comment:
add a new line at the end of the file. github seems to notice it for some reason, in atom when i save a file it auto adds a newline at the end of the file. If you hover over that little icon with the arrow next to it on line 32 it'll show you that message